features/openhab-addons-external/src/main/resources/conf/knx.cfg

```diff
@@ -60,3 +60,7 @@
 # Seconds to wait for an orderly shutdown of the auto refresher's 
 # ScheduledExecutorService. (optional, defaults to 5)
 #scheduledExecutorServiceShutdownTimeoutString=
+
+# Use NAT (Network Address Translation)
+#  (optional; defaults to false)
+#useNAT=
```

Comment:
At the risk of seeming too picky, if this showed the commented-out `true` setting, then it would not be left to assume that `true` was the precise thing to enter here when the user uncomments the line.